src/components/core/MainContent.tsx

```diff
@@ -10,11 +10,17 @@ import { carbonEvents } from 'services/events';
 import { ErrorUserBlocked } from 'components/core/error/ErrorUserBlocked';
 import {
   Outlet,
-  PathNames,
+  Pathnames,
   useRouterState,
   useScrollRestoration,
 } from 'libs/routing';
 
+const paths: Record<string, Pathnames> = {
+  debug: '/debug',
```

Comment:
why do we have path strings everywhere? Having type safety doesn't mean it's a good idea to write hard coded strings in more than one place